# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/man/pl-nlu-10-fix.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./src/man.c.orig	Wed Jun  6 00:01:47 2001
+++ ./src/man.c	Mon Jun 11 00:25:27 2001
@@ -514,7 +514,7 @@
 		 the current position plus 10 lines.  This plus setpl()
 		 gives us a single page that just contains the whole
 		 man page. (William Webber, wew@cs.rmit.edu.au) */
-	      strcat(buft, "; echo; echo \".pl \\n(nlu+10\"");
+	      strcat(buft, "; echo; echo \".pl (n;lu+10\"");
 #if 0
 	      /* In case this doesnt work for some reason,
 		 michaelkjohnson suggests: I've got a simple
